tests/e2e/lib.sh

```diff
@@ -948,6 +948,11 @@ wait_for_api() {
 
     start_time="$(date '+%s')"
     max_seconds=${MAX_WAIT_SECONDS:-300}
+    if [[ "${ORCHESTRATOR_FLAVOR}" == "openshift" ]]; then
```

Comment:
This _should_ be set in most cluster e2e jobs, but just in case I would:
```suggestion
    if [[ "${ORCHESTRATOR_FLAVOR:-}" == "openshift" ]]; then
```